packages/apm/src/integrations/tracing.ts

```diff
@@ -565,6 +620,25 @@ export class Tracing implements Integration {
     return time / 1000;
   }
 
+  /**
+   * Adds debug data to the span
+   */
+  private static _addSpanDebugInfo(span: Span): void {
+    if (global.performance) {
+      span.setData('performance', true);
+      span.setData('performance.timeOrigin', global.performance.timeOrigin);
+      span.setData('performance.now', global.performance.now());
+      // tslint:disable-next-line: deprecation
+      if (global.performance.timing && global.performance.timing.navigationStart) {
```

Comment:
```suggestion
      if (global.performance.timing) {
```

It is somewhat useful to see `undefined` in the debug data, same as we'd see for browsers that don't have `performance.timeOrigin` (Safari's case).